x-pack/plugins/security_solution/public/common/utils/endpoint_alert_check.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { find } from 'lodash/fp';
+import { TimelineEventsDetailsItem } from '../../../common/search_strategy';
+
+export const endpointAlertCheck = ({ data }: { data: TimelineEventsDetailsItem[] | null }) => {
```

Comment:
Do we already have something like this somewhere? @dasansol92 implemented something in detections for showing the ability to create an Event Filter from an Endpoint alert only.

Reply:
Also, can you add some tests to this with various timeline events - sone that are endpoint and others that are not